azext_iot/digitaltwins/_help.py

```diff
@@ -323,12 +339,22 @@ def load_digitaltwins_help():
             --servicebus-policy {servicebus_policy}
             -n {instance_name}
 
-        - name: Adds a ServiceBus Topic endpoint to a target instance using Identity based auth.
+        - name: Adds a ServiceBus Topic endpoint to a target instance using system-assigned identity authentication.
+          text: >
+            az dt endpoint create servicebus --endpoint-name {endpoint_name}
+            --servicebus-resource-group {servicebus_resource_group}
+            --servicebus-namespace {servicebus_namespace}
+            --servicebus-topic {servicebus_topic_name}
+            --system
```

Comment:
We are spelling out almost every parameter maybe `--system` should be `--mi-system-assigned`? Same for `--user` to `--mi-user-assigned` below.